dialogue-target/src/main/java/com/palantir/dialogue/DialogueService.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DialogueService {
```

Comment:
I think I'd prefer `DialogueClient`, but the factory interface naming would overlap with the `dialogue-clients` factory components. Thoughts/ideas/concerns?

Reply:
No strong preference, Service kinda stuck with me for whatever reasons. But if DialogueClientFactory is clashing, then it's clashing.

The generated classes from conjure are "AutoDeserializeConfirmServiceBlocking", so I think DialogueService it is.